### DISCUSSION TITLE
strong-pubsub* migration help for LoopBack4

### DISCUSSION THREAD

**Discussion by vdj69** [loopbackio/loopback-next#7187] [Q&A]
Hi, I'm using Loopback3 and want to migrate to Loopback4 but my application using the below messaging connectors for TCP and MQTT communication but nothing is there in the Loopback 4 document. 

- strong-pubsub
- strong-pubsub-bridge
- strong-pubsub-connection-mqtt

Please help?

**Comment by dhmlau** [comment]
@raymondfeng, could you please help on this question? Thanks.

**Comment by raymondfeng** [comment]
@vdj69 We do see the value to have something similar in LoopBack 4. I started a PoC at https://github.com/strongloop/loopback-next/tree/pubsub/packages/pubsub a while ago but didn't have chances to move it further.

**Comment by vdj69** [comment]
> @vdj69 We do see the value to have something similar in LoopBack 4. I started a PoC at https://github.com/strongloop/loopback-next/tree/pubsub/packages/pubsub a while ago but didn't have chances to move it further.

@raymondfeng 
Please let me know if you need my help, I will create an emulator for your testing which send multiple sensors data via mqtt.
